libs/common/src/lib/util/tm-log.service.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
I think here message can be typed as string or object.

Reply:
Done. Unfortunately, I had to leave the linter rule disabled because of the optionalParams